Delivery-service/Models/Contracts/ICompany.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using Models.Models;
+
+namespace Models.Contracts
+{
+    public interface ICompany
```

Comment:
you should use interfaces for services
not for entities

interface should contain behavior 
Name it is data not behavior